backend/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+/** A user with saved preferences. */
+public final class User {
+
+  /** Datastore ID */
+  private final long id;
+  /** Google username (ldap) */
+  private final String username;
+
+  /** Initialize constructor fields */
+  public User(long id, String username) {
+    // TODO(#32): add User interests
+    this.id = id;
+    this.username = username;
+  }
+
+  public long getId() {
+    return id;
+  }
+
+  public String getUsername() {
+    return username;
+  }
+
+  public String toString() {
+    return "username=" + username + "\n";
```

Comment:
https://guava.dev/releases/20.0/api/docs/com/google/common/base/MoreObjects.ToStringHelper.html is a commonly used class for removing some of the extra boilerplate. If you use that then it shouldn't be too difficult to add the id as well.